.github/workflows/ci.yml

```diff
@@ -110,6 +110,7 @@ jobs:
           - os: ubuntu-24.04
             ocaml_compiler: ocaml-variants.5.2.1+options,ocaml-option-static
             # Reanalyze does not work on OCaml 5.3.0 anymore, therefore run it on 5.2.1
+            # disabled for now, since Reanalyze will stop working going forward. Can likely be removed entirely soon
```

Comment:
Since this is the only use of reanalyze, how about removing it entirely.
No point in bringing it back later, as it would immediately give an error.
CC @cknitt

Reply:
Yes, go ahead!